actionpack/lib/action_controller/metal/rate_limiting.rb

```diff
@@ -49,23 +83,39 @@ module ClassMethods
       #     end
       #
       #     class SessionsController < ApplicationController
-      #       rate_limit to: 3, within: 2.seconds, name: "short-term"
-      #       rate_limit to: 10, within: 5.minutes, name: "long-term"
+      #       rate_limit to: 3.times, within: 2.seconds, name: "short-term"
+      #       rate_limit to: 10.times, within: 5.minutes, name: "long-term"
       #     end
-      def rate_limit(to:, within:, by: -> { request.remote_ip }, with: -> { head :too_many_requests }, store: cache_store, name: nil, **options)
+      def rate_limit(to:, within:, by: :rate_limit_by, with: :rate_limit_with, store: cache_store, name: nil, **options)
+        to = to.size if to.is_a?(Enumerator)
```

Comment:
This change isn't necessary and that API is worse. In no world `rate_limit to: 3.times` is better than `rate_limit to: 3`

Reply:
Ah, just saw Kasper's suggestion. I guess it makes sense, but I really don't like it. Maybe `rate_limit times: 3`? The extra allocation to create an array and the conditional to check if it is an array, makes this syntax sugar not worth for me.